apps/voucherify/package.json

```
@@ -0,0 +1,52 @@
+{
+  "name": "voucherify-contentful-app",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "start": "cross-env BROWSER=none react-scripts start",
+    "postinstall": "npm run build",
+    "build": "react-scripts build",
+    "install-ci": "npm ci",
+    "create-app-definition": "contentful-app-scripts create-app-definition",
+    "upload": "contentful-app-scripts upload --bundle-dir ./build",
+    "upload-ci": "contentful-app-scripts upload --ci --bundle-dir ./build --organization-id $CONTENTFUL_ORG_ID --definition-id $CONTENTFUL_APP_DEF_ID --token $CONTENTFUL_ACCESS_TOKEN",
+    "deploy": "contentful-app-scripts upload --ci --bundle-dir ./build --organization-id ${DEFINITIONS_ORG_ID} --definition-id 6xt1AyXyw6aSnn5JDXaFin --token ${CONTENTFUL_CMA_TOKEN}",
+    "deploy:staging": "contentful-app-scripts upload --ci --bundle-dir ./build --organization-id ${TEST_ORG_ID} --definition-id 6xt1AyXyw6aSnn5JDXaFin --token ${TEST_CMA_TOKEN}"
```

Can you update the app id for this `deploy:staging` script so that it will update our staging app (id: `5kadpWSYKnUZrW8ys6uvkx`)? 

```suggestion
    "deploy:staging": "contentful-app-scripts upload --ci --bundle-dir ./build --organization-id ${TEST_ORG_ID} --definition-id 5kadpWSYKnUZrW8ys6uvkx --token ${TEST_CMA_TOKEN}"
```

---

Should it be the same id (5kadpWSYKnUZrW8ys6) in `deploy`? Or should I leave the id of our custom app there? (qLqU3Zs2d5awizzTcZ1mK)